### DISCUSSION TITLE
Back dating contracts and rent payments

### DISCUSSION THREAD

**Discussion by SteveTeece** [microrealestate/microrealestate#125] [Q&A]
I work for a Disability and Accommodation provider in Australia. We have a small number of rental properties that are sub-let to our clients and staff.

I am trying to enter some historical data, but I get an error if I try to commence a lease before today's date.

Is there a way to back date rental agreements and payments so we have historical data in MRE?

**Comment by camelaissani** [comment]
This will be addressed in #126
